app/main.py

```diff
@@ -73,7 +77,7 @@
         ],
         "weapon": {
             "name": "Sword",
-            "power": 45
+            "power": 55
```

Comment:
why did you edit this?

Reply:
I was trying to find out, if class is calculating data correctly, and forgot to change it back